packages/distributor/index.ts

```diff
@@ -1 +1 @@
-export * as StreamflowDistributorSolana from "./solana";
+export * as StreamflowDistributorSolana from "./solana/index.js";
```

Comment:
you said you were planning to migrate scoped exports here, do I miss anything here?

Reply:
I gave up on that - there are issues with v7 even with that, right now I paused that work in favour of more prio work (aligned unlocks/airdrops)